app/templates/DonationForm.html.twig

```diff
@@ -30,17 +23,21 @@
     {% include 'Form3.html.twig' %}
 </div>
 
-<script>
-
-    // The following line is a Twig template placeholder to generate initial values (defaults to empty object)
-    var initialFormValues = {% if initialFormValues %}{$ initialFormValues|json_encode|raw $}{% else %}{}{% endif %};
-
-    // TODO add initial validation status to show validation errors when coming from the server
-
+<script id="init-form"
+        data-initial-form-values='{% if initialFormValues %}{$ initialFormValues|json_encode|e("html_attr") $}{% else %}{}{% endif %}'
```

Comment:
Quotation is inconsistent, maybe use double quotes for HTML attributes and single quotes for twig calls?